### DISCUSSION TITLE
CyberPatriot Script Discussion

### DISCUSSION THREAD

**Discussion by kxlieannh** [kxlieannh/cpscripts#6] [General]
Use this chat for questions, discussion, script ideas.. etc.

Please keep the chat appropriate and respectful.

**Comment by txmattt** [comment]
Finished up testing all window scripts but winrdp.ps1 just a head ups

**Comment by txmattt** [comment]
😫😫😫

**Comment by kxlieannh** [comment]
I just updated the wiki, im gonna start including details about the scripts.

if your lost on scripts, no your not, look at the wiki.

**Comment by txmattt** [comment]
is the wiki complete so i can check it off

**Reply by kxlieannh** [reply]
no, im actively working on it

**Comment by kxlieannh** [comment]
### Script Ideas (Linux and Win)

- Monitor file integrity
- Set file permissions
- Secure boot settings
- Configure Auto updates
- Enable app armor
- Install configure fail 2 ban
- Secure  prevent shared memory
- Secure USB storage
- Configure log 
- Highlight Security events
- Scan unusual process or network
- log unauthorized connections

**Comment by txmattt** [comment]
made 10 linux scripts

**Comment by txmattt** [comment]
y'all need to update the readme for windows for every script y'all make and test just a fyi.

**Comment by txmattt** [comment]
wont be free till tmw night to make any changes or additions

**Comment by kxlieannh** [comment]
im boutta lock in and do a bunch of thesed windows scripts. if your making scripts pretty please be sure to update the README on that branch so that we are aware of the script status, and also so i can update the wiki.

**Comment by kxlieannh** [comment]
I made the issue scripts into project issues now, if you guys want to review them or start working on issues, feel free to. These are windows scripts issues. If you have a question, leave comments on the issue you are referring to or on the discussion.
